server/model/db.js

```diff
@@ -2,7 +2,7 @@ const {Client} = require('pg');
 const config = require('./config.js');
 const client = new Client(config);
 
-client.query(`set search_path to 'test'`);
+client.query(`set search_path to 'public'`);
```

Comment:
I'm not a Postgres pro..  Does search_path set the schema, the database?  And is public a special name?  Is it just coincidental that we don't really need to specify test, even without these other changes, since we (were) already in the test database?

Reply:
Honestly, this line should not be in the code at all. By default, postgres searches for tables in the public schema. The old line set the schema to look in the test schema which we're not using anymore. We could theoretically leave this line here but it isn't necessary anymore. The client will search in the public schema be default.